cfme/configure/configuration/__init__.py

```diff
@@ -602,8 +602,8 @@ class SMTPSettings(Updateable):
             ('port', Input("smtp_port")),
             ('domain', Input("smtp_domain")),
             ('start_tls', Input("smtp_enable_starttls_auto")),
-            ('ssl_verify', Select("select#smtp_openssl_verify_mode")),
-            ('auth', Select("select#smtp_authentication")),
+            ('ssl_verify', AngularSelect("smtp_openssl_verify_mode")),
```

Comment:
@psav, I experienced some weird error with Select and it disappeared when I switched to AngularSelect. Though those should work almost the same in this case.